src/main/kotlin/org/neo4j/graphql/CrudOperations.kt

```diff
@@ -1,37 +1,36 @@
 package org.neo4j.graphql
 
 import graphql.language.FieldDefinition
-import graphql.language.ListType
 import graphql.language.ObjectTypeDefinition
 
 data class Augmentation(val create: String = "", val merge: String = "", val update: String = "", val delete: String = "",
-                        val inputType: String = "", val ordering: String = "", val filterType: String = "", val query: String = "")
+        val inputType: String = "", val ordering: String = "", val filterType: String = "", val query: String = "")
 
 fun createNodeMutation(ctx: Translator.Context, type: ObjectTypeDefinition): Augmentation {
     val typeName = type.name
     val idField = type.fieldDefinitions.find { it.type.name() == "ID" }
     val scalarFields = type.fieldDefinitions.filter { it.type.isScalar() }.sortedByDescending { it == idField }
-    val has_idField = type.fieldDefinitions.find { it.name.equals("_id") } != null
+    val hasIdField = type.fieldDefinitions.find { it.name == "_id" } != null
     val idFieldArg = idField?.let { it.name + ":" + it.type.render() }
 
     val result = if (ctx.mutation.enabled && !ctx.mutation.exclude.contains(typeName) && scalarFields.isNotEmpty()) {
-        val fieldArgs = scalarFields.map { it.name + ":" + it.type.render() }.joinToString(", ")
+        val fieldArgs = scalarFields.joinToString(", ") { it.name + ":" + it.type.render() }
```

Comment:
I don't like this style at all it makes it harder to understand what's happening. As joining comes after the mapping not before.